src/mlagility/cli/cli.py

```diff
@@ -297,11 +328,35 @@ def main():
     )
 
     #######################################
-    # Parser for the "delete" command
+    # Parser for the "cache print" command
+    #######################################
+
+    print_parser = cache_subparsers.add_parser(
```

Comment:
Is `status` a better representation of this function than `print`?

Reply:
I’m fine with that, or `info` or similar. @danielholanda ?

Reply:
Or `stats`!

Reply:
I'm going with stats.